<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IgnoreFilesDialog</class>
 <widget class="QDialog" name="IgnoreFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ignore Files</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>511</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="ignore_files_line_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
  </widget>
  <widget class="QListWidget" name="ignore_files_list_widget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>441</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ignore_files_remove_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ignore_files_add_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ignore_files_update_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="ignore_files_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ignored Files &amp; Directories - Must include the full path</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>IgnoreFilesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>IgnoreFilesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
